tools/README.md

```diff
@@ -2,21 +2,21 @@
 
 ### TheGraph integration
 
-|             |   Status    |
-| ----------- | ----------- |
+|             |   Status    |   Description    |
+| ----------- | ----------- | ----------- |
 | Catch & handle events emitted from EthereumTransactions | ✅ |
 | Catch & handle events emitted from ContractCall & ContractCreate | ✅ |
 | Capture ERC transfers through HTS precompile | ✅ |
 | Execute contract calls during subgraph event processing | ✅ |
 | Being able to specify the startBlock from which the graph sync | ✅ |
 | Support for multiple dataSources | ✅ |
+| Support for dynamic dataSource templates | ✅ |
 | Block Handlers WITHOUT filters | ✅ |
-| Block Handlers WITH filters | ❌ |
-| Call Handlers | ❌ |
-| Capture HTS transfers through HTS precompile | ❌ |
-| Capture HTS token transfers through HAPI | ❌ |
-| Can use dynamic dataSource templates | ❔ |
-| Can index anonymous events | ❔ |
+| Block Handlers WITH filters | ❌ | Requires Parity's [trace_filter](https://openethereum.github.io/JSONRPC-trace-module#trace_filter)
+| Call Handlers | ❌ | Requires Parity's [trace_filter](https://openethereum.github.io/JSONRPC-trace-module#trace_filter)
+| Capture HTS transfers through HTS precompile | ❌ | Depends on [4127](https://github.com/hashgraph/hedera-services/issues/4127)
+| Capture HTS token transfers through HAPI | ❌ | Depends on [4337](https://github.com/hashgraph/hedera-mirror-node/issues/4337), [4738](https://github.com/hashgraph/hedera-mirror-node/issues/4738)
+| Can index anonymous events | ❔ | Has not been tested
```

Comment:
Can you turn this into a relay ticket to explore indexing of anonymous events?

Reply:
Opened an Issue to investigate if anonymous events work on Hedera and are handled correctly

Reply:
Yeah just update the doc with that issue as you have with the others so it's easy to view.